example/src/App.tsx

```diff
@@ -152,6 +161,24 @@ const App = () => {
     AppLovinMAX.addEventListener('OnBannerAdCollapsedEvent', () => {
       logStatus('Banner ad collapsed')
     });
+
+    // MREC Ad Listeners
+    AppLovinMAX.addEventListener('OnMRecAdLoadedEvent', () => {
```

Comment:
Might be better semantics if we declare `adInfo` between the `()`.

Reply:
Only adUnitIdentifier is passed in the adInfo : https://github.com/AppLovin/AppLovin-MAX-React-Native/blob/bcff0c8cf4087e30ce1530ce7868f34d85b5581e/ios/AppLovinMAX.m#L527

Reply:
Yeah but what if we do this: 

```
AppLovinMAX.addEventListener('OnMRecAdLoadedEvent', (adInfo) => {
```